content/en/profiler/enabling/ddprof.md

```diff
@@ -217,26 +195,15 @@ You can configure logging to one of several endpoints:
 - `disable` disables the logs entirely.
 - Any other value is treated as a file path, with a leading `/` designating an absolute path.
 
-### Pidmode
-
-When a target PID is given, instrument that PID. When this mode is engaged, any parameter after typical profiler argument processing is ignored. If the owning UID of the target PID and the profiler differ, do one of the following:
-- Run the profiler as the root process instead.
-- Grant `CAP_PERFMON` to the profiler (Linux v5.8 or later).
-- Grant `CAP_SYS_ADMIN` to the profiler.
-- Decrease `perf_event_paranoid` to `-1` (consult your distro documentation or system administrator for details).
-
-When a PID is specified in this way, only child processes (both forks and threads) spawned after instrumentation will be followed.
-
-### Globalmode
+### Global
 
-Global mode is intended for debug purposes. When `global` is set to `yes`, the profiler attempts to profile all the visible processes. 
-This requires elevated permissions (for example, running as root or granting `CAP_PERFMON`, `CAP_SYSADMIN`) or setting `perf_event_paranoid` to `-1`.
+If you want to instrument all running process, you can try out the `--global` option.
+Global mode is intended for debug purposes. This requires elevated permissions. Depending on your setup, this can mean running as root, granting `CAP_PERFMON`, `CAP_SYSADMIN`, or setting `perf_event_paranoid` to `-1`.
```

Comment:
```suggestion
To instrument all running process, use the `--global` option. Global mode is intended for debug purposes and requires elevated permissions. Depending on your setup, this can mean running as root, granting `CAP_PERFMON`, `CAP_SYSADMIN`, or setting `perf_event_paranoid` to `-1`.
```